### ISSUE TITLE
`plot_cast_ts` not working for NA or PI

### ISSUE THREAD

**Issue by ethanwhite** [weecology/portalcasting#200]
```R
> plot_cast_ts(data_set = "controls", species = toupper("NA"))
Error in `[.data.frame`(obs, , c("moon", species)) : 
  undefined columns selected

> plot_cast_ts(data_set = "controls", species = toupper("PI"))
Error in `[.data.frame`(obs, , c("moon", species)) : 
  undefined columns selected
```

**Comment by juniperlsimonis** [comment]
@ethanwhite are you still having this issue or was it resolved in the updates?

**Comment by henrykironde** [comment]
I think I have installed the latest portalcasting from devtools::install_github("weecology/portalcasting")
But I still have the same issue.

**Comment by ethanwhite** [comment]
I'm seeing the same thing as @henrykironde after updating both the package and all of the data.

**Comment by juniperlsimonis** [comment]
i'm adding an error catch that tells you what's happening (there's not a column of that species in the cast table)

the PI failure is because we, by default ("base" settings) don't model PI, so there aren't cast data. 
`portalcasting::rodent_species(set = "all")`
`portalcasting::rodent_species(set = "base")`

the NA failure is an NA hiccup again... it will never end... but should be patched in v0.21.1, which i'm releasing rn
